### PR TITLE
Another stab at 2.5.1 

### PR DESCRIPTION
**Pull request by nrooney**

### CODE REVIEW

**Comment by dwsinger** [comment]
LGTM. we should highlight to the AC that we might make this clearer.

**Comment by innnzzz6** [comment]
251abtag

**Comment by innnzzz6** [comment]
gh-pages

**Comment by frivoal** [comment]
As this is editorial, I think describing it in a Disposition of Comments sent together with the AC vote would be enough, but if you want, we could also send a short mail as a reply to the review request that's been already send, pointing to this PR and saying we'll merge in the next week as long as nobody has an issue with it.

File: index.html [hunk 1]
```diff
@@ -763,16 +763,15 @@ <h4 id="AB-TAG-constraints">2.5.1 Advisory Board and Technical Architecture Grou
           At the completion of the next regularly scheduled
           election for the TAG, the organization <em class="rfc2119">must</em> have returned to having at most
           one seat.</li>
-        <li>Two participants with the same primary affiliation per <a href="#AB-TAG-elections">2.5.2</a> must not both occupy elected seats on the AB.</li>
+        <li>Two participants with the same primary affiliation per <a href="#AB-TAG-elections">2.5.2</a> must not both occupy 		elected seats on the AB. If, for whatever reason, these constraints are not satisfied
```
Comment (xfq) [inline index.html]: Would you please remove the two tab characters in the middle of this line? Thank you!

**Comment by frivoal** [comment]
> LGTM with nits

Nits fixed in https://github.com/w3c/w3process/pull/231/commits/d1fade3a02f7ac54102c9a1deafb17d7cb861754

**Comment by frivoal** [comment]
CfC in favor of merging this documented here: https://lists.w3.org/Archives/Public/public-w3process/2018Dec/0013.html

**Pull request merged**